ros2interface/ros2interface/verb/show.py

```diff
@@ -139,7 +186,15 @@ def add_arguments(self, parser, cli_name):
         arg.completer = type_completer
 
     def main(self, *, args):
+
+        is_show_top_level_comments = args.all_comments or not args.no_comments
```

Comment:
I think the this can be simplified further, right?
```suggestion
        is_show_top_level_comments = not args.no_comments
```